test/widgets/compose_box_test.dart

```diff
@@ -405,98 +407,102 @@ void main() {
       matching: find.widgetWithIcon(IconButton, icon));
 
     void checkComposeBoxParts({required bool areShown}) {
-      check(contentFieldFinder().evaluate().length).equals(areShown ? 1 : 0);
+      final inputFieldCount = inputFieldFinder().evaluate().length;
+      areShown ? check(inputFieldCount).isGreaterThan(0) : check(inputFieldCount).equals(0);
```

Comment:
This looks like an interesting change (`contentFieldFinder` -> `inputFieldFinder`), because this checks if the length is `> 0` now. Looks like this is for handling the presence of the topic input. We may move this to the next commit where it becomes necessary.